### PR TITLE
Use tini in dask-notebook

### PR DESCRIPTION
**Pull request by mrocklin**
cc @jakirkham

### CODE REVIEW

**Comment by mrocklin** [comment]
So, this still doesn't respond to Ctrl-C when I execute it with `docker run`.  Is this a sign that I'm not using `tini` properly?

File: notebook/Dockerfile [hunk 1]
```diff
@@ -12,4 +12,5 @@ RUN chmod +x /usr/bin/prepare.sh
 RUN mkdir /opt/app
 USER $NB_USER
 
-CMD ["bash", "-c", "/usr/bin/prepare.sh && start.sh jupyter lab"]
+ENTRYPOINT ["tini", "--", "bash", "/usr/bin/prepare.sh"]
```
Comment (jakirkham) [inline notebook/Dockerfile]: So we need to download it. There is a conda-forge package. Alternatively can use prebuilt binaries from the author.

Comment (jakirkham) [reply notebook/Dockerfile]: Also I guess we still need `-c` here.

Comment (mrocklin) [reply notebook/Dockerfile]: I think that it is in the base-notebook image https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile#L92

Comment (jakirkham) [reply notebook/Dockerfile]: Ah sorry, we are using the base-notebook as a parent image. Missed that. NVM then.

**Comment by jakirkham** [comment]
Looks like this will fix issue ( https://github.com/dask/dask-docker/issues/4 ) too.

**Comment by jakirkham** [comment]
> So, this still doesn't respond to Ctrl-C when I execute it with `docker run`.  Is this a sign that I'm not using `tini` properly?

Sorry somehow missed your question when looking at this. Would try `docker run -it`.

File: notebook/Dockerfile (hunk 1 above)
Comment (jakirkham) [reply notebook/Dockerfile]: We might be able to get away without `bash` here actually.

**Comment by mrocklin** [comment]
Removed bash.  This behaves as I would expect.

**Comment by jakirkham** [comment]
Glad to hear it. FWIW LGTM.

**Comment by jakirkham** [comment]
We might want to do the same with `base` for consistency.

**Pull request merged**